WordPress/src/main/java/org/wordpress/android/viewmodel/posts/PostListViewModel.kt

```diff
@@ -660,6 +725,62 @@ class PostListViewModel @Inject constructor(
         }
     }
 
+    private fun loadPostFromRemote(localPostId: Int) {
+        // We need network connection to load a remote post
+        if (!checkNetworkConnection()) {
+            return
+        }
+
+        val post = postStore.getPostByLocalPostId(localPostId)
+        if (post != null) {
+            originalPostCopyForConflictUndo = post.clone()
+            dispatcher.dispatch(PostActionBuilder.newFetchPostAction(RemotePostPayload(post, site)))
+            _toastMessage.postValue(ToastMessageHolder(R.string.toast_conflict_updating_post, Duration.SHORT))
+        } else {
+            localPostIdForConflictResolutionDialog = null
+        }
+    }
+
+    private fun savePostFromLocal(localPostId: Int) {
+        // We need network connection to push local version to remote
+        if (!checkNetworkConnection()) {
+            return
+        }
+
+        val post = postStore.getPostByLocalPostId(localPostId)
+        if (post == null) {
+            return
+        }
+
+        localPostIdForConflictResolutionDialog = null
+        // keep a copy for undoing
+        originalPostCopyForConflictUndo = post.clone()
+
+        // we make remoteLastModified match local lastModified to clear the "conflicted" flag in the Post list only
+        post.remoteLastModified = post.lastModified
+        dispatcher.dispatch(PostActionBuilder.newUpdatePostAction(post))
+
+        // and now show a snackbar, acting as if the Post was pushed, but effectively push it after the snackbar is gone
+        var isUndoed = false
+        val undoAction = {
+            isUndoed = true
+            // no op, fake
```

Comment:
Similar to my earlier comment, I don't think this is actually considered a `no-op` and I am not sure what `fake` means in this context.

Reply:
agreed, removed in 80abdf7b51